.cargo/config

```diff
@@ -1,8 +1,8 @@
+[alias]
+xtask = "run -p xtask --"
+
 [target.'cfg(all(target_arch = "arm", target_os = "none"))']
 runner = 'arm-none-eabi-gdb'
 rustflags = [
     "-C", "link-arg=-Tlink.x",
 ]
-
-[build]
-target = "thumbv7em-none-eabihf"
```

Comment:
why is this gone? I would assume this is still relevant for development?

Reply:
Sort of. We now explicitly pass the right target in CI, and I've found this default to be a bit confusing since it doesn't work on nRF51, the nRF52810, and the nRF9160.

Demos can add their own `.cargo/config` if they want though, that should make development easier.

Reply:
Ah, I guess it does work on the nRF9160, but it's not usually the target you *want* to use there.

Reply:
Hmm ok, so I guess you suggest just using the build script during development I guess :) I always hit cargo build out of habit =D but the change & its reasoning is fair enough for me!

Reply:
Yeah, you can still do that if you `cd` to the demo project you're editing and add a `.cargo/config` there.

Reply:
I don't care about the demo projects ;) I care about building the HAL if I change something (which hasn't happened in a log time).

Reply:
The HALs should all build even for x86_64 (at least that's how docs.rs generates documentation for them)

Reply:
Does cortex-m really build for x86 targets?

Reply:
Yes, it goes out of its way to ensure that